### ISSUE TITLE
Refresh button does not work

### ISSUE THREAD

**Issue by MM-Mikhail** [emartisoft/DAControlGUI#7]
As said Refresh button may not work and crash the system into reboot.

Open DAGUI
mount some adf using WB icon
Refresh list in GUI -> BOOM

**Comment by emartisoft** [comment]
It works on my system A600 and A1200 (real and WinUAE)

**Comment by MM-Mikhail** [comment]
Yes, if you are lucky it will but will crash eventually anyway. 
Try running the tool with Enforcer, it will not have a chance of even starting due to memory trashing.
